--- "BD PadeGest.sql"	2019-11-11 21:08:49.151737100 +0100
+++ "BD PadeGest.sql.new"	2019-11-11 21:09:45.977400200 +0100
@@ -579,9 +579,10 @@ END$$
 DELIMITER ;
-SET SQL_MODE = '';
-DROP USER IF EXISTS PadeGestApp;
-SET SQL_MODE='';
-CREATE USER 'PadeGestApp' IDENTIFIED BY 'PadeGestApp';
 
+-- -----------------------------------------------------
+-- User PadeGestApp
+-- -----------------------------------------------------
+DROP USER IF EXISTS 'PadeGestApp';
+CREATE USER 'PadeGestApp' IDENTIFIED BY 'PadeGestApp';
 GRANT TRIGGER, UPDATE, SELECT, INSERT, INDEX, DELETE, ALTER, REFERENCES, DROP, CREATE ON TABLE PADEGEST.* TO 'PadeGestApp';
-GRANT EXECUTE ON function `PADEGEST`.`reservaQueOcupaPista` TO 'PadeGestApp';
+GRANT EXECUTE ON FUNCTION PADEGEST.reservaQueOcupaPista TO 'PadeGestApp';
 
