--- "BD PadeGest.sql"	2019-11-05 16:11:07.454468100 +0100
+++ "BD PadeGest.sql.new"	2019-11-05 16:13:24.505179300 +0100
@@ -549,13 +549,10 @@ END$$
 DELIMITER ;
-SET SQL_MODE = '';
-DROP USER IF EXISTS PadeGestApp;
-SET SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
-CREATE USER 'PadeGestApp' IDENTIFIED BY 'PadeGestApp';
 
+-- -----------------------------------------------------
+-- User PadeGestApp
+-- -----------------------------------------------------
+DROP USER IF EXISTS 'PadeGestApp';
+CREATE USER 'PadeGestApp' IDENTIFIED IDENTIFIED WITH mysql_native_password BY 'PadeGestApp';
 GRANT TRIGGER, UPDATE, SELECT, INSERT, INDEX, DELETE, ALTER, REFERENCES, DROP, CREATE ON TABLE PADEGEST.* TO 'PadeGestApp';
 
-SET SQL_MODE=@OLD_SQL_MODE;
-SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS;
-SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS;
-
 -- -----------------------------------------------------
